packages/terra-detail-view/src/DetailView.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import DetailViewSubtitle from './DetailViewSubtitle';
+import './DetailView.scss';
+
+const propTypes = {
+  /**
+   * The title display for the view
+   **/
+  title: PropTypes.element,
+
+  /**
+   * List of subtitle displays
+   **/
+  subtitles: PropTypes.arrayOf(PropTypes.element),
+
+  /**
+   * Display for visualization data
+   **/
+  graph: PropTypes.element,
+
+  /**
+   * Array of elements to display detail information
+   **/
+  details: PropTypes.arrayOf(PropTypes.element),
+
+  /**
+   * Text to be displayed at the footer of the view
+   **/
+  footer: PropTypes.string,
+
+  /**
+   * Indicates if sections should be devided
+   **/
+  divided: PropTypes.bool,
```

Comment:
The standard for booleans have been to prefix with `is`. `isDivided`. I agree that divided reads a lot nicer, but the is prefix does immediately indicate type.